packages/grafana-ui/src/components/PanelChrome/PanelChrome.test.tsx

```diff
@@ -152,31 +152,35 @@ it('collapses the controlled panel when user clicks on the chevron or the title'
 
   expect(screen.getByText("Panel's Content")).toBeInTheDocument();
 
-  const button = screen.getByText('Default title');
+  const button = screen.getByRole('button', { name: 'Default title' });
   // collapse button should have same aria-controls as the panel's content
-  expect(button.getAttribute('aria-controls')).toBe(button.parentElement?.parentElement?.nextElementSibling?.id);
+  expect(button.getAttribute('aria-controls')).toBe(
+    button.parentElement?.parentElement?.parentElement?.nextElementSibling?.id
+  );
 
   fireEvent.click(button);
 
   expect(screen.queryByText("Panel's Content")).not.toBeInTheDocument();
   // aria-controls should be removed when panel is collapsed
   expect(button).not.toHaveAttribute('aria-controlls');
-  expect(button.parentElement?.parentElement?.nextElementSibling?.id).toBe(undefined);
+  expect(button.parentElement?.parentElement?.parentElement?.nextElementSibling?.id).toBe(undefined);
```

Comment:
Not a fan of these very specific nested test selectors, maybe there's another way to match these elements? Maybe giving the expanded div a test-id or aria-role if it's appropriate there?

Reply:
Good point, these may break easily otherwise